readme.md

```diff
@@ -606,6 +606,7 @@
 - [Food](https://github.com/jzarca01/awesome-food#readme) - Food-related projects on GitHub.
 - [Mental Health](https://github.com/dreamingechoes/awesome-mental-health#readme) - Mental health awareness and self-care in the software industry.
 - [Bitcoin Payment Processors](https://github.com/alexk111/awesome-bitcoin-payment-processors#readme) - Start accepting Bitcoin.
+- [Product Design](https://github.com/teoga/awesome-product-design) — Links and articles to guide digital product designers.
```

Comment:
> The entry in the Awesome list should:
> * Include a short description about the project/theme of the list. **It should not describe the list itself.**

Reply:
Would something like this work? 
"Helpful resources for digital product designer."

Reply:
That still describes the list. It should describe what a digital product design/designer is.

Reply:
I just changed it to this
"Research, design, test, learn, iterate."